src/app/settings/keyboard-mapping-settings/components/edit-keyboard-mapping-dialog/edit-keyboard-mapping-dialog.component.html

```diff
@@ -0,0 +1,37 @@
+<sofie-sidebar-container [title]="getTitle()">
+    <sofie-row class="mb-3">
+        <form [formGroup]="actionTriggerForm" class="left-container">
+            <form [formGroup]="actionTriggerDataForm">
```

Comment:
I wonder if it is better to use `formGroupName="groupName"` instead 2 forms. That way the form can be one and the structure can be nested in this form.

Reply:
I originally didn't because I didn't want to add an extra div. But I have changed it to use `formGroupName`.